sympy/core/function.py

```diff
@@ -875,6 +875,16 @@ def _diff_wrt(self):
         return True
 
 
+class UndefSage(object):
+    def __get__(self, ins, typ):
+        from sage.calculus.var import function
+        if ins is None:
+            return lambda: function(typ.__name__)
+        else:
+            args = [arg._sage_() for arg in ins.args]
+            return lambda : function(ins.__class__.__name__)(*args)
```

Comment:
The code in this method doesn't appear to be tested according to coverage measurement. When would this code be invoked?